src/st/wrp/e_misc.c

```diff
@@ -83,7 +83,7 @@ u16 g_eUnk14SpawnRots[] = {
 
 #include "../entity_unkId15_spawner.h"
 
-u16 D_80180FF8[] = {
+s16 unk15_rot[] = {
```

Comment:
One small change here. We tend to call global variables with the prefix `g_` (so in this case `g_Unk15_rot`). That way it's clearer if a variable is either local or global when reading functions.

The `g_` thing does not apply whenever we can mark the variable as `static`. This is a great opportunity to mark the array as `static. as it looks like it is not used anywhere else! No need to do anything it in other overlays as it can give us a clear indicator in the future that `unk15_rot` can be self-contained in any of the C files the data will be imported to.